dashboard/modules/state/state_head.py

```diff
@@ -38,7 +38,8 @@ def _options_from_req(self, req) -> ListApiOptions:
         # Only apply 80% of the timeout so that
         # the API will reply before client times out if query to the source fails.
         timeout = int(req.query.get("timeout"))
-        return ListApiOptions(limit=limit, timeout=timeout)
+        filters = eval(req.query.get("filters", "[]"))
```

Comment:
instead of evals, I used this pattern (https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#query-parameter-list-multiple-values). Seems like it is the best practice (basically receive filter_Keys and filter_values and reconstruct a list of tuple)